### ISSUE TITLE
SSH never connects as part of the build process

### ISSUE THREAD

**Issue by maworsham** [canonical/packer-maas#261]
I am trying to follow the instructions per https://maas.io/docs/how-to-build-an-ubuntu-image and need to have it use my [user-data-lvm.txt](https://github.com/user-attachments/files/16910571/user-data-lvm.txt) template (see attached) instead of the one that comes with the GitHub repo. I have tried to build the ISO with the original code as well as mine, and it just keeps timing out on the SSH part of the build.

The SSH part just goes on and on and never connects. I've done in in 3 and 5 minute intervals and still never connects. I am doing this on a local server - both as a LXD VM and also tried it as a VMware VM and it has the same conclusion.

```
ubuntu@juppddpckr001:~/packer-maas/ubuntu$ make custom-ubuntu-lvm.dd.gz
rm -f -rf output-* custom-*.gz \
        seeds-flat.iso seeds-lvm.iso seeds-cloudimg.iso \
        OVMF_VARS.fd
cloud-localds seeds-lvm.iso user-data-lvm meta-data
cp -v /usr/share/OVMF/OVMF_VARS_4M.fd OVMF_VARS.fd
'/usr/share/OVMF/OVMF_VARS_4M.fd' -> 'OVMF_VARS.fd'
packer init . && packer build -only=qemu.lvm \
        -var ubuntu_series=jammy \
        -var ubuntu_lvm_iso=ubuntu-22.04.4-live-server-amd64.iso \
        -var architecture=amd64 \
        -var ovmf_suffix= \
        -var timeout=1h .
Installed plugin github.com/hashicorp/qemu v1.1.0 in "/home/ubuntu/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64"
qemu.lvm: output will be in this color.

==> qemu.lvm: Retrieving ISO
==> qemu.lvm: Trying https://releases.ubuntu.com/jammy/ubuntu-22.04.4-live-server-amd64.iso
==> qemu.lvm: Trying https://releases.ubuntu.com/jammy/ubuntu-22.04.4-live-server-amd64.iso?checksum=sha256%3A45f873de9f8cb637345d6e66a583762730bbea30277ef7b32c9c3bd6700a32b2
    qemu.lvm: ubuntu-22.04.4-live-server-amd64.iso 1.96 GiB / 1.96 GiB [============================================================================================================] 100.00% 1m16s
==> qemu.lvm: https://releases.ubuntu.com/jammy/ubuntu-22.04.4-live-server-amd64.iso?checksum=sha256%3A45f873de9f8cb637345d6e66a583762730bbea30277ef7b32c9c3bd6700a32b2 => packer_cache/jammy.iso
==> qemu.lvm: Starting HTTP server on port 8115
==> qemu.lvm: Found port for communicator (SSH, WinRM, etc): 3600.
==> qemu.lvm: Creating temporary RSA SSH key for instance...
==> qemu.lvm: Looking for available port between 5900 and 6000 on 127.0.0.1
==> qemu.lvm: Starting VM, booting from CD-ROM
    qemu.lvm: The VM will be run headless, without a GUI. If you want to
    qemu.lvm: view the screen of the VM, connect via VNC without a password to
    qemu.lvm: vnc://127.0.0.1:5977
==> qemu.lvm: Overriding default Qemu arguments with qemuargs template option...
==> qemu.lvm: Waiting 2s for boot...
==> qemu.lvm: Connecting to VM via VNC (127.0.0.1:5977)
==> qemu.lvm: Typing the boot commands over VNC...
    qemu.lvm: Not using a NetBridge -- skipping StepWaitGuestAddress
==> qemu.lvm: Using SSH communicator to connect: 127.0.0.1
==> qemu.lvm: Waiting for SSH to become available...
```

**Comment by alanbach** [comment]
Hello @maworsham What version of packer do you have installed? I have seen this issue on older versions...

**Comment by maworsham** [comment]
> Hello @maworsham What version of packer do you have installed? I have seen this issue on older versions...

ubuntu@juppddpckr001:~$ packer --version
1.8.6

**Comment by alanbach** [comment]
Strange! I was on 1.8.6 until a couple of days ago and this exact build worked for me (My machine is 24.04 LTS).

Try adding FOREGROUND=1 PACKER_LOG=1 in front of make to see if it uncovers more details.

**Comment by maworsham** [comment]
This the output with the above information added to the 'make' command...

```
ubuntu@juppddpckr001:~/packer-maas/ubuntu$ FOREGROUND=1 PACKER_LOG=1 make custom-ubuntu-lvm.dd.gz
rm -f -rf output-* custom-*.gz \
        seeds-flat.iso seeds-lvm.iso seeds-cloudimg.iso \
        OVMF_VARS.fd
cloud-localds seeds-lvm.iso user-data-lvm meta-data
cp -v /usr/share/OVMF/OVMF_VARS_4M.fd OVMF_VARS.fd
'/usr/share/OVMF/OVMF_VARS_4M.fd' -> 'OVMF_VARS.fd'
packer init . && packer build -only=qemu.lvm \
        -var ubuntu_series=jammy \
        -var ubuntu_lvm_iso=ubuntu-22.04.4-live-server-amd64.iso \
        -var architecture=amd64 \
        -var ovmf_suffix= \
        -var timeout=1h .
2024/09/06 17:41:54 [INFO] Packer version: 1.8.6 [go1.18.9 linux amd64]
2024/09/06 17:41:54 [TRACE] discovering plugins in /usr/local/bin
2024/09/06 17:41:54 [TRACE] discovering plugins in /home/ubuntu/.config/packer/plugins
2024/09/06 17:41:54 [DEBUG] Discovered plugin: qemu = /home/ubuntu/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64
2024/09/06 17:41:54 [INFO] found external [-packer-default-plugin-name-] builders from qemu plugin
2024/09/06 17:41:54 [TRACE] discovering plugins in .
2024/09/06 17:41:54 [INFO] PACKER_CONFIG env var not set; checking the default config file path
2024/09/06 17:41:54 [INFO] PACKER_CONFIG env var set; attempting to open config file: /home/ubuntu/.packerconfig
2024/09/06 17:41:54 [WARN] Config file doesn't exist: /home/ubuntu/.packerconfig
2024/09/06 17:41:54 [INFO] Setting cache directory: /home/ubuntu/.cache/packer
2024/09/06 17:41:54 [TRACE] init: plugingetter.ListInstallationsOptions{FromFolders:[]string{"/usr/local/bin/packer", ".", "/home/ubuntu/.config/packer/plugins"}, BinaryInstallationOptions:plugingetter.BinaryInstallationOptions{APIVersionMajor:"5", APIVersionMinor:"0", OS:"linux", ARCH:"amd64", Ext:"", Checksummers:[]plugingetter.Checksummer{plugingetter.Checksummer{Type:"sha256", Hash:(*sha256.digest)(0xc000106900)}}}}
2024/09/06 17:41:54 [TRACE] listing potential installations for "github.com/hashicorp/qemu" that match "~> 1.0". plugingetter.ListInstallationsOptions{FromFolders:[]string{"/usr/local/bin/packer", ".", "/home/ubuntu/.config/packer/plugins"}, BinaryInstallationOptions:plugingetter.BinaryInstallationOptions{APIVersionMajor:"5", APIVersionMinor:"0", OS:"linux", ARCH:"amd64", Ext:"", Checksummers:[]plugingetter.Checksummer{plugingetter.Checksummer{Type:"sha256", Hash:(*sha256.digest)(0xc000106900)}}}}
2024/09/06 17:41:54 [TRACE] for plugin github.com/hashicorp/qemu found 1 matching installation(s)
2024/09/06 17:41:54 [INFO] (telemetry) Finalizing.
2024/09/06 17:41:54 waiting for all plugin processes to complete...
2024/09/06 17:41:55 [INFO] Packer version: 1.8.6 [go1.18.9 linux amd64]
2024/09/06 17:41:55 [TRACE] discovering plugins in /usr/local/bin
2024/09/06 17:41:55 [TRACE] discovering plugins in /home/ubuntu/.config/packer/plugins
2024/09/06 17:41:55 [DEBUG] Discovered plugin: qemu = /home/ubuntu/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64
2024/09/06 17:41:55 [INFO] found external [-packer-default-plugin-name-] builders from qemu plugin
2024/09/06 17:41:55 [TRACE] discovering plugins in .
2024/09/06 17:41:55 [INFO] PACKER_CONFIG env var not set; checking the default config file path
2024/09/06 17:41:55 [INFO] PACKER_CONFIG env var set; attempting to open config file: /home/ubuntu/.packerconfig
2024/09/06 17:41:55 [WARN] Config file doesn't exist: /home/ubuntu/.packerconfig
2024/09/06 17:41:55 [INFO] Setting cache directory: /home/ubuntu/.cache/packer
2024/09/06 17:41:55 [TRACE] listing potential installations for "github.com/hashicorp/qemu" that match "~> 1.0". plugingetter.ListInstallationsOptions{FromFolders:[]string{"/usr/local/bin/packer", ".", "/home/ubuntu/.config/packer/plugins"}, BinaryInstallationOptions:plugingetter.BinaryInstallationOptions{APIVersionMajor:"5", APIVersionMinor:"0", OS:"linux", ARCH:"amd64", Ext:"", Checksummers:[]plugingetter.Checksummer{plugingetter.Checksummer{Type:"sha256", Hash:(*sha256.digest)(0xc0007aca00)}}}}
2024/09/06 17:41:55 [TRACE] Found the following "github.com/hashicorp/qemu" installations: [{/home/ubuntu/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 v1.1.0}]
2024/09/06 17:41:55 [INFO] found external [-packer-default-plugin-name-] builders from qemu plugin
2024/09/06 17:41:55 [TRACE] validateValue: not active for flat_filename, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for http_proxy, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for ubuntu_series, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for customize_script, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for architecture, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for ubuntu_iso, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for ssh_username, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for ssh_ubuntu_password, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for filename, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for kernel, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for no_proxy, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for timeout, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for ubuntu_lvm_iso, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for https_proxy, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for http_directory, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for ssh_password, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for ovmf_suffix, so skipping
2024/09/06 17:41:55 [TRACE] validateValue: not active for headless, so skipping
2024/09/06 17:41:55 [TRACE] Starting external plugin /home/ubuntu/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 start builder -packer-default-plugin-name-
2024/09/06 17:41:55 Starting plugin: /home/ubuntu/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 []string{"/home/ubuntu/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64", "start", "builder", "-packer-default-plugin-name-"}
2024/09/06 17:41:55 Waiting for RPC address for: /home/ubuntu/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64
2024/09/06 17:41:55 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:41:55 Plugin address: unix /tmp/packer-plugin3135405995
2024/09/06 17:41:55 Received unix RPC address for /home/ubuntu/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64: addr is /tmp/packer-plugin3135405995
2024/09/06 17:41:55 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:41:55 Waiting for connection...
2024/09/06 17:41:55 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:41:55 Serving a plugin connection...
2024/09/06 17:41:55 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:41:55 [TRACE] starting builder -packer-default-plugin-name-
2024/09/06 17:41:55 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:41:55 use detected accelerator: tcg
2024/09/06 17:41:55 [TRACE] Starting internal plugin packer-provisioner-file
2024/09/06 17:41:55 Starting plugin: /usr/local/bin/packer []string{"/usr/local/bin/packer", "plugin", "packer-provisioner-file"}
2024/09/06 17:41:55 Waiting for RPC address for: /usr/local/bin/packer
2024/09/06 17:41:55 packer-provisioner-file plugin: [INFO] Packer version: 1.8.6 [go1.18.9 linux amd64]
2024/09/06 17:41:55 packer-provisioner-file plugin: [INFO] PACKER_CONFIG env var not set; checking the default config file path
2024/09/06 17:41:55 packer-provisioner-file plugin: [INFO] PACKER_CONFIG env var set; attempting to open config file: /home/ubuntu/.packerconfig
2024/09/06 17:41:55 packer-provisioner-file plugin: [WARN] Config file doesn't exist: /home/ubuntu/.packerconfig
2024/09/06 17:41:55 packer-provisioner-file plugin: [INFO] Setting cache directory: /home/ubuntu/.cache/packer
2024/09/06 17:41:55 packer-provisioner-file plugin: args: []string{"packer-provisioner-file"}
2024/09/06 17:41:55 packer-provisioner-file plugin: Plugin address: unix /tmp/packer-plugin3842871979
2024/09/06 17:41:55 packer-provisioner-file plugin: Waiting for connection...
2024/09/06 17:41:55 Received unix RPC address for /usr/local/bin/packer: addr is /tmp/packer-plugin3842871979
2024/09/06 17:41:55 packer-provisioner-file plugin: Serving a plugin connection...
2024/09/06 17:41:55 [TRACE] Starting internal plugin packer-provisioner-shell
2024/09/06 17:41:55 Starting plugin: /usr/local/bin/packer []string{"/usr/local/bin/packer", "plugin", "packer-provisioner-shell"}
2024/09/06 17:41:55 Waiting for RPC address for: /usr/local/bin/packer
2024/09/06 17:41:56 packer-provisioner-shell plugin: [INFO] Packer version: 1.8.6 [go1.18.9 linux amd64]
2024/09/06 17:41:56 packer-provisioner-shell plugin: [INFO] PACKER_CONFIG env var not set; checking the default config file path
2024/09/06 17:41:56 packer-provisioner-shell plugin: [INFO] PACKER_CONFIG env var set; attempting to open config file: /home/ubuntu/.packerconfig
2024/09/06 17:41:56 packer-provisioner-shell plugin: [WARN] Config file doesn't exist: /home/ubuntu/.packerconfig
2024/09/06 17:41:56 packer-provisioner-shell plugin: [INFO] Setting cache directory: /home/ubuntu/.cache/packer
2024/09/06 17:41:56 packer-provisioner-shell plugin: args: []string{"packer-provisioner-shell"}
2024/09/06 17:41:56 Received unix RPC address for /usr/local/bin/packer: addr is /tmp/packer-plugin3952461875
2024/09/06 17:41:56 packer-provisioner-shell plugin: Plugin address: unix /tmp/packer-plugin3952461875
2024/09/06 17:41:56 packer-provisioner-shell plugin: Waiting for connection...
2024/09/06 17:41:56 packer-provisioner-shell plugin: Serving a plugin connection...
2024/09/06 17:41:56 [TRACE] Starting internal plugin packer-post-processor-compress
2024/09/06 17:41:56 Starting plugin: /usr/local/bin/packer []string{"/usr/local/bin/packer", "plugin", "packer-post-processor-compress"}
2024/09/06 17:41:56 Waiting for RPC address for: /usr/local/bin/packer
2024/09/06 17:41:56 packer-post-processor-compress plugin: [INFO] Packer version: 1.8.6 [go1.18.9 linux amd64]
2024/09/06 17:41:56 packer-post-processor-compress plugin: [INFO] PACKER_CONFIG env var not set; checking the default config file path
2024/09/06 17:41:56 packer-post-processor-compress plugin: [INFO] PACKER_CONFIG env var set; attempting to open config file: /home/ubuntu/.packerconfig
2024/09/06 17:41:56 packer-post-processor-compress plugin: [WARN] Config file doesn't exist: /home/ubuntu/.packerconfig
2024/09/06 17:41:56 packer-post-processor-compress plugin: [INFO] Setting cache directory: /home/ubuntu/.cache/packer
2024/09/06 17:41:56 packer-post-processor-compress plugin: args: []string{"packer-post-processor-compress"}
2024/09/06 17:41:56 Received unix RPC address for /usr/local/bin/packer: addr is /tmp/packer-plugin2397328234
2024/09/06 17:41:56 packer-post-processor-compress plugin: Plugin address: unix /tmp/packer-plugin2397328234
2024/09/06 17:41:56 packer-post-processor-compress plugin: Waiting for connection...
2024/09/06 17:41:56 packer-post-processor-compress plugin: Serving a plugin connection...
qemu.lvm: output will be in this color.

2024/09/06 17:41:56 Build debug mode: false
2024/09/06 17:41:56 Force build: false
2024/09/06 17:41:56 On error:
2024/09/06 17:41:56 Waiting on builds to complete...
2024/09/06 17:41:56 Starting build run: qemu.lvm
2024/09/06 17:41:56 Running builder:
2024/09/06 17:41:56 [INFO] (telemetry) Starting builder qemu.lvm
2024/09/06 17:41:56 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:41:56 Qemu path: /usr/bin/qemu-system-x86_64, Qemu Image path: /usr/bin/qemu-img
==> qemu.lvm: Retrieving ISO
==> qemu.lvm: Trying https://releases.ubuntu.com/jammy/ubuntu-22.04.4-live-server-amd64.iso
==> qemu.lvm: Trying https://releases.ubuntu.com/jammy/ubuntu-22.04.4-live-server-amd64.iso?checksum=sha256%3A45f873de9f8cb637345d6e66a583762730bbea30277ef7b32c9c3bd6700a32b2
2024/09/06 17:41:56 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:41:56 Acquiring lock for: https://releases.ubuntu.com/jammy/ubuntu-22.04.4-live-server-amd64.iso?checksum=sha256%3A45f873de9f8cb637345d6e66a583762730bbea30277ef7b32c9c3bd6700a32b2 (packer_cache/jammy.iso.lock)
==> qemu.lvm: https://releases.ubuntu.com/jammy/ubuntu-22.04.4-live-server-amd64.iso?checksum=sha256%3A45f873de9f8cb637345d6e66a583762730bbea30277ef7b32c9c3bd6700a32b2 => packer_cache/jammy.iso
2024/09/06 17:42:02 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:02 Leaving retrieve loop for ISO
2024/09/06 17:42:02 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:02 No floppy files specified. Floppy disk will not be made.
2024/09/06 17:42:02 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:02 No CD files specified. CD disk will not be made.
2024/09/06 17:42:02 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:02 [INFO] Creating disk with Path: output-lvm/packer-lvm and Size: 8G
2024/09/06 17:42:02 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:02 Executing qemu-img: []string{"create", "-f", "raw", "output-lvm/packer-lvm", "8G"}
2024/09/06 17:42:02 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:02 stdout: Formatting 'output-lvm/packer-lvm', fmt=raw size=8589934592
2024/09/06 17:42:02 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:02 stderr:
2024/09/06 17:42:02 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:02 Found available port: 8328 on IP: 0.0.0.0
==> qemu.lvm: Starting HTTP server on port 8328
==> qemu.lvm: Found port for communicator (SSH, WinRM, etc): 2711.
==> qemu.lvm: Creating temporary RSA SSH key for instance...
2024/09/06 17:42:02 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:02 Looking for available communicator (SSH, WinRM, etc) port between 2222 and 4444
2024/09/06 17:42:02 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:02 Found available port: 2711 on IP: 127.0.0.1
==> qemu.lvm: Looking for available port between 5900 and 6000 on 127.0.0.1
2024/09/06 17:42:05 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:05 Looking for available port between 5900 and 6000 on 127.0.0.1
2024/09/06 17:42:05 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:05 Found available port: 5916 on IP: 127.0.0.1
2024/09/06 17:42:05 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:05 Found available VNC port: 5916 on IP: 127.0.0.1
2024/09/06 17:42:05 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:05 Qemu --version output: QEMU emulator version 6.2.0 (Debian 1:6.2+dfsg-2ubuntu6.22)
2024/09/06 17:42:05 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: Copyright (c) 2003-2021 Fabrice Bellard and the QEMU Project developers
2024/09/06 17:42:05 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:05 Qemu version: 6.2.0
==> qemu.lvm: Starting VM, booting from CD-ROM
    qemu.lvm: view the screen of the VM, connect via VNC without a password to
    qemu.lvm: vnc://127.0.0.1:5916
2024/09/06 17:42:05 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:05 Qemu Builder has no floppy files, not attaching a floppy.
2024/09/06 17:42:05 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:05 Executing /usr/bin/qemu-system-x86_64: []string{"-m", "2048M", "-boot", "once=d", "-name", "packer-lvm", "-vnc", "127.0.0.1:16", "-vga", "qxl", "-device", "virtio-blk-pci,drive=drive0,bootindex=0", "-device", "virtio-blk-pci,drive=cdrom0,bootindex=1", "-device", "virtio-blk-pci,drive=drive1,bootindex=2", "-device", "virtio-net,netdev=user.0", "-drive", "if=pflash,format=raw,id=ovmf_code,readonly=on,file=/usr/share/OVMF/OVMF_CODE.fd", "-drive", "if=pflash,format=raw,id=ovmf_vars,file=OVMF_VARS.fd", "-drive", "file=output-lvm/packer-lvm,if=none,id=drive0,cache=writeback,discard=ignore,format=raw", "-drive", "file=seeds-lvm.iso,format=raw,cache=none,if=none,id=drive1,readonly=on", "-drive", "file=packer_cache/jammy.iso,if=none,id=cdrom0,media=cdrom", "-netdev", "user,id=user.0,hostfwd=tcp::2711-:22", "-machine", "type=pc,accel=tcg", "-smp", "2"}
    qemu.lvm: The VM will be run headless, without a GUI. If you want to
    qemu.lvm: view the screen of the VM, connect via VNC without a password to
    qemu.lvm: vnc://127.0.0.1:5916
==> qemu.lvm: Overriding default Qemu arguments with qemuargs template option...
2024/09/06 17:42:05 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:05 Started Qemu. Pid: 11448
==> qemu.lvm: Waiting 2s for boot...
==> qemu.lvm: Connecting to VM via VNC (127.0.0.1:5916)
2024/09/06 17:42:09 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:09 Connected to VNC desktop: QEMU (packer-lvm)
==> qemu.lvm: Typing the boot commands over VNC...
2024/09/06 17:42:09 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:09 [INFO] Waiting 1s
2024/09/06 17:42:10 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:10 Sending char 'e', code 0x65, shift false
2024/09/06 17:42:10 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:10 [INFO] Waiting 5s
2024/09/06 17:42:15 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:15 Special code '<down>' found, replacing with: 0xFF54
2024/09/06 17:42:16 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:16 [INFO] Waiting 1s
2024/09/06 17:42:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:17 Special code '<down>' found, replacing with: 0xFF54
2024/09/06 17:42:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:17 [INFO] Waiting 1s
2024/09/06 17:42:18 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:18 Special code '<down>' found, replacing with: 0xFF54
2024/09/06 17:42:18 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:18 [INFO] Waiting 2s
2024/09/06 17:42:20 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:20 Special code '<end>' found, replacing with: 0xFF57
2024/09/06 17:42:20 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:20 [INFO] Waiting 5s
2024/09/06 17:42:25 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:25 Special code '<bs>' found, replacing with: 0xFF08
2024/09/06 17:42:25 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:25 Special code '<bs>' found, replacing with: 0xFF08
2024/09/06 17:42:26 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:26 Special code '<bs>' found, replacing with: 0xFF08
2024/09/06 17:42:26 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:26 Special code '<bs>' found, replacing with: 0xFF08
2024/09/06 17:42:26 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:26 [INFO] Waiting 1s
2024/09/06 17:42:27 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:27 Sending char 'a', code 0x61, shift false
2024/09/06 17:42:27 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:27 Sending char 'u', code 0x75, shift false
2024/09/06 17:42:27 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:27 Sending char 't', code 0x74, shift false
2024/09/06 17:42:28 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:28 Sending char 'o', code 0x6F, shift false
2024/09/06 17:42:28 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:28 Sending char 'i', code 0x69, shift false
2024/09/06 17:42:28 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:28 Sending char 'n', code 0x6E, shift false
2024/09/06 17:42:28 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:28 Sending char 's', code 0x73, shift false
2024/09/06 17:42:28 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:28 Sending char 't', code 0x74, shift false
2024/09/06 17:42:29 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:29 Sending char 'a', code 0x61, shift false
2024/09/06 17:42:29 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:29 Sending char 'l', code 0x6C, shift false
2024/09/06 17:42:29 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:29 Sending char 'l', code 0x6C, shift false
2024/09/06 17:42:29 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:29 Sending char ' ', code 0x20, shift false
2024/09/06 17:42:30 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:30 Sending char '-', code 0x2D, shift false
2024/09/06 17:42:30 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:30 Sending char '-', code 0x2D, shift false
2024/09/06 17:42:30 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:30 Sending char '-', code 0x2D, shift false
2024/09/06 17:42:30 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:30 [INFO] Waiting 1s
2024/09/06 17:42:31 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:31 Special code '<f10>' found, replacing with: 0xFFC7
    qemu.lvm: Not using a NetBridge -- skipping StepWaitGuestAddress
==> qemu.lvm: Using SSH communicator to connect: 127.0.0.1
==> qemu.lvm: Waiting for SSH to become available...
2024/09/06 17:42:31 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:31 [INFO] Waiting for SSH, up to timeout: 1h0m0s
2024/09/06 17:42:31 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:31 [INFO] Attempting SSH connection to 127.0.0.1:2711...
2024/09/06 17:42:31 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:31 [DEBUG] reconnecting to TCP connection for SSH
2024/09/06 17:42:31 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:42:31 [DEBUG] handshaking with SSH
2024/09/06 17:43:31 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:43:31 [DEBUG] SSH handshake err: Timeout during SSH handshake
2024/09/06 17:43:38 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:43:38 [INFO] Attempting SSH connection to 127.0.0.1:2711...
2024/09/06 17:43:38 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:43:38 [DEBUG] reconnecting to TCP connection for SSH
2024/09/06 17:43:38 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:43:38 [DEBUG] handshaking with SSH
```

The following part just cycles over and over...
```
2024/09/06 17:43:38 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:43:38 [INFO] Attempting SSH connection to 127.0.0.1:2711...
2024/09/06 17:43:38 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:43:38 [DEBUG] reconnecting to TCP connection for SSH
2024/09/06 17:43:38 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:43:38 [DEBUG] handshaking with SSH
2024/09/06 17:44:38 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/06 17:44:38 [DEBUG] SSH handshake err: Timeout during SSH handshake
```
